staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -409,10 +409,52 @@ func (qs *queueSet) syncTimeLocked() {
 	realNow := qs.clock.Now()
 	timeSinceLast := realNow.Sub(qs.lastRealTime)
 	qs.lastRealTime = realNow
-	qs.currentR += SeatsTimesDuration(qs.getVirtualTimeRatioLocked(), timeSinceLast)
+	prevR := qs.currentR
```

Comment:
Can we move this commit to a separate PR?

I'm fine with that conceptually, but I think it requires a bit more attention.
And I would really like to unblock the other PRs (which are blocked on the other two commits) first.

Reply:
Yes.

Reply:
See #105469